pkg/agent/metrics/prometheus.go

```diff
@@ -67,6 +67,24 @@ var (
 		Help:           "Flow count for each OVS flow table. The TableID is used as a label.",
 		StabilityLevel: metrics.STABLE,
 	}, []string{"table_id"})
+
+	OVSFlowOpsErrorCount = metrics.NewCounterVec(
```

Comment:
do you think that `OVSFlowOpsCount` would also be a good metrics to have?

Reply:
Yes, it sounds good. I'll work on it. Thank you for your suggestion.